patches/@metamask+controllers+30.0.1.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/@metamask/controllers/dist/util.js b/node_modules/@metamask/controllers/dist/util.js
+index 574c57b..d12dd57 100644
+--- a/node_modules/@metamask/controllers/dist/util.js
++++ b/node_modules/@metamask/controllers/dist/util.js
+@@ -835,6 +835,7 @@ var SupportedTokenDetectionNetworks;
+     SupportedTokenDetectionNetworks["bsc"] = "56";
+     SupportedTokenDetectionNetworks["polygon"] = "137";
+     SupportedTokenDetectionNetworks["avax"] = "43114";
++    SupportedTokenDetectionNetworks["local"] = "1337";
```

Comment:
this is a temporary patch. Going to patch `@metmask/controllers` and cut `v30.0.2` and pull it in asap.